config_loader.go

```diff
@@ -8,7 +8,7 @@ import (
 	"os"
 	"path/filepath"
 
-	"gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v3"
```

Comment:
wasn't obvious before, but we were carrying `v2` and `v3` of the yaml module. We're consolidating on `v3` here since its the LCD